Makefile

```diff
@@ -89,9 +89,14 @@ $(TARGET): $(OBJS)
 	@echo "AS " $@
 	@ $(CC) -c -o $@ $(AFLAGS) $<
 
+DEPFLAGS := -MMD
```

Comment:
The flag is already part of COMMON_FLAGS.

Reply:
Fun. Then we only need the rule to actually include those dep files.